students/Gorthi_kavita/9module/9assignment.py

```diff
@@ -0,0 +1,134 @@
+class donar():
```

Comment:
You have no `__init__` function for all of your classes.  Do some more reading on defining Classes to get an idea what what its suppose to look like.